servers/rendering/renderer_rd/cluster_builder_rd.cpp

```diff
@@ -47,6 +47,15 @@ ClusterBuilderSharedDataRD::ClusterBuilderSharedDataRD() {
 	}
 
 	{
+		// FIXME: this block of code causes crash on tvOS
```

Comment:
Not sure how this should be approached - by skipping this shader compilation at all or using `RenderingDevice::FramebufferFormatID`